shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitConstants.java

```diff
@@ -14,7 +14,13 @@ public final class ShortCircuitConstants {
     private ShortCircuitConstants() {
     }
 
-    public static final boolean SUBTRANS_STUDY = false;
+    public enum StudyType {
+        SUB_TRANSIENT,
+        TRANSIENT,
+        STEADY_STATE
+    }
 
+    public static final StudyType DEFAULT_STUDY_TYPE = StudyType.TRANSIENT;
     public static final boolean WITH_FEEDER_RESULT = true;
```

Comment:
This boolean can be removed, except if we choose to have a global parameter that will be a priority over the boolean we have inside each fault description.